enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/OperationApproveRequest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.enrollmentserver.impl.service;
+
+import com.wultra.core.http.common.request.RequestContext;
+import io.getlime.security.powerauth.crypto.lib.enums.PowerAuthSignatureTypes;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.util.List;
+
+/**
+ * Parameter object for {@link MobileTokenService#operationApprove(OperationApproveRequest)}.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@Builder
+@Getter
+public class OperationApproveRequest {
```

Comment:
There are two classes in the project with the same name `OperationApproveRequest`. Is this a good idea?

Reply:
Nice catch, renamed to `OperationApproveParameterObject`.